docs/deployment.md

```diff
@@ -5,7 +5,11 @@ Example Kubernetes manifests are provided in [/examples/kubernetes](/examples/ku
 1. Generate TLS certs [/docs/tls.md](/docs/tls.md) and update [/examples/kubernetes/mutating-webhook-configuration.yaml](/examples/kubernetes/mutating-webhook-configuration.yaml) with the `caBundle`
 2. Update [/examples/kubernetes/deployment.yaml](/examples/kubernetes/deployment.yaml) with the appropriate version you want to deploy
 3. Specify whatever flags you want in the deployment.yaml
-4. Create ConfigMaps (or sidecar config files on disk somewhere) so the injector has some sidecars to inject :) [/docs/configmaps.md](/docs/configmaps.md)
+4. Create a kubernetes secret from the certificates that you generated as a part of [/docs/tls.md](/docs/tls.md).
+```
+kubectl create secret generic k8s-sidecar-injector-bf2-production --from-file=examples/tls/us-east-1/PRODUCTION/sidecar-injector.crt --from-file=examples/tls/us-east-1/PRODUCTION/sidecar-injector.key --namespace=kube-system
```

Comment:
Would you mind updating the secret name here (and in https://github.com/tumblr/k8s-sidecar-injector/blob/master/examples/kubernetes/deployment.yaml#L26) to be named `k8s-sidecar-injector` instead? The `-bf2-production` is a bit of leftover stuff I forgot to remove when opensourcing :)

Also, this copy should definitely reflect that the `--from-file` paths should point to your actual `$AZ/$CLUSTER/sidecar-injector.{key,crt}` files, not `bf2` or `us-east-1`

Reply:
Got it :)